### ISSUE TITLE
lavPredict(..., se = TRUE) error with missing = "FIML"

### ISSUE THREAD

**Issue by TDJorgensen** [yrosseel/lavaan#156]
This came to my attention when you answered this question on the lavaan forum:

https://groups.google.com/d/msg/lavaan/RmMWGsfj_eQ/4OnP5UEbBQAJ

Do the `se` and `acov` attributes also need to be created per missing-data pattern?

```
data("HolzingerSwineford1939")
HolzingerSwineford1939$x1[1] <- NA
HS.model <- ' visual  =~ x1 + x2 + x3
              textual =~ x4 + x5 + x6
              speed   =~ x7 + x8 + x9 '
fit <- cfa(HS.model, data = HolzingerSwineford1939, missing = "fiml")

lavPredict(fit)              # no problem
lavPredict(fit, acov = TRUE) # ERROR
```

**Comment by yrosseel** [comment]
The error is now fixed. But the fact remains that the SEs are not computed yet in the fiml case. I will try to fix this soon (albeit not for 0.6-5). Instead of a vector (as in the complete case), I will compute SEs for each observation and return a matrix. 
As for the ACOVs, there is indeed a different ACOV per missing pattern. Not sure if we should return these. (The same is true for fsm; currently, only a 'global' one is returned).

**Comment by TDJorgensen** [comment]
> there is indeed a different ACOV per missing pattern. Not sure if we should return these.

`plausibleValues()` would need them, but I will just add a warning when missing patterns are detected, advising users they can use multiple imputations instead of FIML to get plausible values from a `lavaan.mi` object.  

> The same is true for fsm; currently, only a 'global' one is returned

By the time `fsm()` and `sam()` are ready for public consumption, `plausibleValues()` will probably be obsolete anyway, so maybe this is not something to worry about in the long term.

**Comment by yrosseel** [comment]
Fixed in lavaan dev 0.6-6.1542. When missing = "ml", acov (and fsm) are now returned per missing pattern. The standard errors are returned as a matrix with the SEs for each observation.
